### ISSUE TITLE
XXNET导致内存溢出

### ISSUE THREAD

**Issue by LuoboWang1** [XX-net/XX-Net#4442]
可以正常使用但是会经常导致内存溢出， 看了chrome的任务管理器发现时V8代理插件内存占用飙升。正常运行只占4G内存的百分之五十左右，而V8代理进程可以占用内存达到1.5G以上 直接挤爆内存。然后黑屏重启。
![image](https://cloud.githubusercontent.com/assets/20898507/19726084/acec5680-9bbb-11e6-879e-451917aeb2dc.png)
![image](https://cloud.githubusercontent.com/assets/20898507/19726100/bd3476b2-9bbb-11e6-9f1d-cbcc48e6158b.png)

**Comment by LuoboWang1** [comment]
![image](https://cloud.githubusercontent.com/assets/20898507/19726358/276e8f4e-9bbd-11e6-8965-8643b90c6b83.png)
就想这样子 然后黑屏直接ip归零 重新扫描。

**Comment by LuoboWang1** [comment]
![screenshot_20161026204451](https://cloud.githubusercontent.com/assets/20898507/19726408/65fac192-9bbd-11e6-868e-521e0ac37ebf.png)

**Comment by zhencaihao** [comment]
扫描开太高的关系？默认是50你这个是500.

**Comment by LuoboWang1** [comment]
不是 我开到10也是一样的。我只有把XXnet关了然后把Omega移除才可不出现崩溃状况。刚刚更新版本的时候从2.6到2.7开始用还好，但是后面就经常溢出。

2016-10-26 22:10 GMT+08:00, zhencaihao <notifications@github.com>:

**Comment by nly4lov** [comment]
我连啥叫内存溢出都不知道，也不知道内存堆栈。会不会是楼主内存太小了呀？

**Comment by jzp820927** [comment]
估计有可能是插件，或者 chrome 的问题吧

**Comment by wenliyang** [comment]
Chrome的bug，换浏览器

**Comment by LuoboWang1** [comment]
chrome上的插件比较多 暂时还不准备换。再有一个就是原来3.19版本一切正常但是更新到3,26以后就出现崩溃状况。现在更新到2.2.7就好一点了。

2016-10-27 9:24 GMT+08:00 wenliyang <notifications@github.com>:

**Comment by kkst17** [comment]
https://github.com/XX-net/XX-Net/issues/3847
试试
